site/content/agent-app-widgets/advanced-use.md

```diff
@@ -6,44 +6,197 @@ weight: 30
 
 ## Developing your own widget
 
-If you want to build your own widget, be sure to include both the [LiveChat Boilerplate](/docs/boilerplate) and [JavaScript Widget API](#javascript-widgets-api):
+If you're building your own widget and you need to interact with the Agent App, be sure to use the [Agent App SDK](#agent-app-sdk).
 
-> Our widget SDK package is hosted on NPM. You can get it with following command:
+If you've already developed a widget, you're probably using the [Widget SDK](#widget-sdk). Keep in mind that **it is no longer actively maintained**. However, we provide backward compatibility for this SDK, so your existing widgets should still be working properly. When building new widgets, we encourage you to use the [Agent App SDK](#agent-app-sdk) instead.
 
+## Accessing LiveChat data
+
+You can leverage the OAuth2.0 authorization flow to access data from the [REST API](/docs/rest-api). Head to [Sign in with LiveChat](/docs/sign-in-with-livechat) docs for more information.
+
+## Layout and Styling
+
+If you're using React, you can use the [LiveChat Design System](https://github.com/livechat/design-system), which is a component library used directly in the LiveChat user interface.
+
+We also ship a [LiveChat Boilerplate](/docs/boilerplate) – it's a lightweight CSS stylesheet to help you lift off with creating the widget interface.
+
+> Place this tag within the `<head></head>` section:
+
+```html
+<link rel="stylesheet" href="//cdn.livechatinc.com/boilerplate/1.1.css">
 ```
-npm install --save @livechat/agent-app-widget-sdk
+
+## Hosting the widget
+
+You can host your widget locally or on a dedicated server. The hosted content has to be served over the **HTTPS Protocol**. 
+
+During development, you can use a self-signed certificate for `localhost` or upload your widget to an SSL-enabled host. You can also leverage bundlers like [Webpack](https://webpack.js.org/configuration/dev-server/) to use an https-enabled development server.
```

Comment:
We link to the recent blogpost from Ola: https://developers.livechatinc.com/blog/widget-hosting-on-netlify/.

Reply:
✅